primereact-toast/app/routes/actions/addProduct.tsx

```diff
@@ -0,0 +1,38 @@
+import { json } from '@remix-run/node';
+
+export async function action({ request }) {
+  let formData = await request.formData();
+  //Valores
```

Comment:
Let's set this in English

```suggestion
  // Values
```